### PR TITLE
Thouship/testing

### PR DESCRIPTION
**Pull request by thouship**

### CODE REVIEW

File: .DS_Store
Comment (stefantan18) [inline .DS_Store:1]: Could delete this.

File: NewsFeedApp/.DS_Store
Comment (stefantan18) [inline NewsFeedApp/.DS_Store:1]: Could delete this.

**Review by stefantan18 — APPROVED** [review]
Looks fine. Maybe could get rid of the .DS_Store files as those are unnecessary.

**Review by stefantan18 — CHANGES_REQUESTED** [review]
Looks fine. Could get rid of the .DS_Store files.

**Comment by aimran1** [comment]
Tests look good and seem to cover all elements of the html page. Agree with Stefan, you should remove the  .DS_Store files.

**Comment by erchardsamoh** [comment]
yep, the tests u've made here should cover most of the facets of the homepage. just need the article cards to be made/merged so you could make the tests for that.

**Comment by jjames003** [comment]
The genre options in the navbar don't need tests as of right now, because we have not given functionality to them. Currently, we have the NewsFeed logo, Preferences, Search, Sign Up, and Log In buttons up and functioning.

**Review by erchardsamoh — APPROVED** [review]
looks good now

**Pull request merged**